<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="mainButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Modern No. 20</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hit!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gambleButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>300</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Gamble!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Save
and
Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumMoney">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Money        0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumResources">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Money        0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unbreakButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QPushButton" name="speedButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upgrade</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnbreak">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Unbreaking: lv.0

Next level
10k money
2 iron</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpeed">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed: lv.0

Next level
10k money
2 gold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="accButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed Up!</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAcc">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>152</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceleration 10x

Consume
1 ruby</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushUnsaveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit
without
Saving</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
